invenio_search_ui/assets/js/invenio_react_search_ui/App.js

```diff
@@ -0,0 +1,119 @@
+/*
+ * This file is part of Invenio.
+ * Copyright (C) 2020 CERN.
+ *
+ * Invenio is free software; you can redistribute it and/or modify it
+ * under the terms of the MIT License; see LICENSE file for more details.
+ */
+
+import React, { Component } from "react";
+import { Container, Grid } from "semantic-ui-react";
+import {
+  ReactSearchKit,
+  SearchBar,
+  BucketAggregation,
+  EmptyResults,
+  Error,
+  ResultsLoader,
+  withState
+} from "react-searchkit";
+import { Results } from "./Results";
+import { InvenioSearchApi } from "react-searchkit";
+
+import "semantic-ui-css/semantic.min.css";
+
+const OnResults = withState(Results);
+
+const resultsPerPageValues = [
```

Comment:
This should be also configurable, depending on the default that the user sets in `RECORDS_REST_DEFAULT_RESULTS_SIZE`

Reply:
I removed this component in the end, as the variable allows only one option